### DISCUSSION TITLE
Automatically Backup To Google Drive or Even Locally

### DISCUSSION THREAD

**Discussion by gangradem** [iSoron/uhabits#1430] [Feature Requests]
Hi,

I would love it if I could access my habits data without having to manually export it every couple of days.

I am trying to create a web interface to go along with this app. I will make it available to everyone for free if anyone is looking for something like that... :)

Let mee know if something like this is already available...

**Comment by LordJon** [comment]
Not sure if it's what you're looking for but an synchronisation function is currently being built. It was planned for 2.0 but for security reasons got postponed to 2.2.

**Reply by PellelNitram** [reply]
@LordJon Do you have any information on what functionality will be included in the synchronisation function you mention above?

**Reply by LordJon** [reply]
As far as I know it will be synchronisation between multiple devices. The exact implementation is not known to me but I do know that you can fibd some topucs about it here on the Github.
